sketch-client/src/index.ts

```diff
@@ -0,0 +1,56 @@
+/**
+* Copyright 2018-present Ampersand Technologies, Inc.
+*/
+
+import { AccountID, BackingContext, SketchBackendInterface, UserInfo } from '../lib/index';
+
+import * as ObjUtils from 'amper-utils/dist/objUtils';
+
+let DataStore;
```

Comment:
For future commit: I don't think there's really a good reason to inject DataStore here anymore. I think I just didn't have the data-store repo ready yet when I did this.

Reply:
I agree. I wanna change a bunch of this to use fewer globals and define more stuff on the sketch class itself as well, which is my next PR.